randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -1713,18 +1713,28 @@ Extra - asset_id: collision_camera_019
       Screw Attack
   > Event - Grapple Block (grapplepulloff1x2_000)
       All of the following:
-          Morph Ball
+          Grapple Beam and Morph Ball
           Any of the following:
               Cross Bomb
               All of the following:
                   # Bomb & Power Bombs, to desync the tunnel bomb blocks from the ledge bomb blocks
                   Bomb and Power Bombs
+              All of the following:
+                  # Power bombs (video: https://twitter.com/i/status/1545111396760690692 )
+                  Power Bombs ≥ 2
+              All of the following:
+                  # Bombs + flash shift (video: https://www.youtube.com/watch?v=NdVesJ6F_6A )
+                  Bomb and Flash Shift
   > Tile Group (WEIGHT)
       Morph Ball
   > Tunnel to Energy Recharge Station Secret
       All of the following:
           Morph Ball and After Ferenia - grapplepulloff1x2_000
-          Cross Bomb or Power Bombs
+          Any of the following:
+              Cross Bomb or Power Bombs ≥ 3
+              All of the following:
+                  # bomb + flash ( video: https://www.youtube.com/watch?v=NdVesJ6F_6A )
+                  Bomb and Flash Shift
```

Comment:
This can also probably have "Use Spin Boost" as an option rather than just Flash Shift. You'll have enough time to get back to the tunnel after the first block gets destroyed (and therefore does not need this video as no trick is involved)

Reply:
By using walljump it is also possible to destroy this block, see video https://twitter.com/i/status/1545679535546372098.
What skill level for walljump should be added for this?
I should also check if the grapple block can be destroyed with spin boost (it is unlikely to do so with wall jumps).